test/e2e/specs/specs-playwright/wp-likes__post-spec.js

```diff
@@ -57,33 +62,20 @@ describe( DataHelper.createSuiteTitle( 'Likes (Post)' ), function () {
 		it( 'Unlike post', async function () {
 			await publishedPostPage.unlikePost();
 		} );
-	} );
-
-	describe( 'Like an existing post', function () {
-		let publishedPostPage;
-
-		it( 'Log in', async function () {
-			const loginFlow = new LoginFlow( page, 'gutenbergSimpleSiteUser' );
-			await loginFlow.logIn();
-		} );
 
-		it( 'Visit site', async function () {
-			const myHomePage = await MyHomePage.Expect( page );
-			await myHomePage.visitSite();
+		it( 'Clear cookies', async function () {
+			await BrowserManager.clearCookies( page );
 		} );
 
-		it( 'Click on first post', async function () {
-			const publishedPostsListPage = await PublishedPostsListPage.Expect( page );
-			await publishedPostsListPage.visitPost( 1 );
-		} );
-
-		it( 'Like post', async function () {
+		it( `Like post as ${ anotherUser }`, async function () {
 			publishedPostPage = await PublishedPostPage.Expect( page );
-			await publishedPostPage.likePost();
-		} );
 
-		it( 'Unlike post', async function () {
-			await publishedPostPage.unlikePost();
+			const loginFlow = new LoginFlow( page, anotherUser );
+
+			// Clicking the Like button will bring up a new popup.
+			// `loginFromPopup` will observe for a popup event, grab the new popup and
+			// execute the login process on that page.
+			await Promise.all( [ loginFlow.logInFromPopup(), publishedPostPage.likePost() ] );
```

Comment:
This step is copied entirely over from the now-deleted file `test/e2e/specs/specs-playwright/wp-likes__logged-out-spec.js`.